js/initcontext.go

```diff
@@ -165,15 +165,30 @@ func (i *InitContext) compileImport(src, filename string) (*goja.Program, error)
 }
 
 func (i *InitContext) Open(name string, args ...string) (goja.Value, error) {
-	filename := loader.Resolve(i.pwd, name)
+	filename := name
+	if filename[0] != '/' && filename[0] != filepath.Separator && !filepath.IsAbs(filename) {
```

Comment:
All of those are because of ... windows ... :( 
I was absolutely certain we will need the '/' for in memory FS to work on windows, but this ... appears to not be the case ... maybe. Like at least in appveyor I think it works ... but ultimately I will need to try it on VM or something to be certain. But this certainly makes the test ,... easier to write :)

The filepath.Separator actually should be '\\' as this is what I care about (no idea why I left it like this). And this has to do with the fact `filepath.IsAbs` is apparently buggy and doesn't recognises `\Program Files\Custom Utilities\StringFinder.exe` as an [absolute path](https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats). Will someone use this ? probably not ... I don't think many people will use absolute paths with `open()`

I am not particularly sure how it works even with all this fixes if someone makes an archive on windows and decides to give it to someone on Linux and vice versa. To be perfectly honest I would prefer if all our paths need to be with '/' only and we don't support absolute paths :) in `open()`. 
Technically I will argue for complete support we need the case with '\\' and the case with '/' makes it more cross platform as ... everybody can just use '/' (golang makes this possible) and your paths will "work" on both windows and sane OSes on the other hand ... this cross platformism seems to only be useful in our tests  ...